rotkehlchen/exchanges/bitcoinde.py

```diff
@@ -57,8 +62,14 @@ def bitcoinde_asset(asset: str) -> Asset:
 
 
 def bitcoinde_pair_to_world(pair: str) -> Tuple[Asset, Asset]:
-    tx_asset = bitcoinde_asset(pair[:3])
-    native_asset = bitcoinde_asset(pair[3:])
+    if len(pair) == 6:
+        tx_asset = bitcoinde_asset(pair[:3])
+        native_asset = bitcoinde_asset(pair[3:])
+    elif len(pair) in (7, 8):
+        tx_asset = bitcoinde_asset(pair[:4])
+        native_asset = bitcoinde_asset(pair[4:])
+    else:
+        raise ValueError(f'Invalid pair: {pair}')
```

Comment:
This ValueError will need to be caught and handled. And we don't raise ValueErrors in the codebase as it's too generic an error. Please turn this into a `DeserializationError`.

 Looking at the code again I think that a good place would be where `trade_from_bitcoinde` is called. And there you should also catch any other deserialization errors that may happen.Such as any stray `KeyError`. If any of them happen just skip the trade with a warning just like we do in other exchanges. Example:
 
 https://github.com/rotki/rotki/blob/4f7bdbe40d1e6738a51ff592657022e4ca4f651c/rotkehlchen/exchanges/kraken.py#L637-L667